spec/models/ems_event_spec.rb

```diff
@@ -476,4 +476,44 @@
       end
     end
   end
+
+  context 'manager refresh' do
+    let(:ems_azure) { FactoryBot.create(:ems_azure) }
```

Comment:
I'd prefer not calling out to specific provider plugin classes, would be better to use core `ems_cloud`/`ems_infra` types

Reply:
@agrare Updated!